src/components/AmountForm.tsx

```diff
@@ -250,6 +252,13 @@ function AmountForm(
                     }
                     textInput.current = ref;
                 }}
+                selection={selection}
```

Comment:
@BrtqKr Please help to explain why we need to change this file

Reply:
I think I might have been missing something from the textinput version, but I might have fixed it along. It's probably no longer necessary